packages/atlas-experiment-table/src/TableCell.js

```diff
@@ -1,19 +1,36 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 import URI from 'urijs'
 
 import { Table, Text } from 'evergreen-ui'
 
 const ANNDATA_EXPERIMENT = `E-ANND`
+
+const ExperimentIconDiv = styled.div`
+  background-color: ${props => props.background};
+  color: ${props => props.color};
+  border-radius: 50%;
+  font-size: 16px;
+  height: 20px;
+  width: 20px;
+  text-align: center;
+  vertical-align: middle;
+  margin-left: 4px;
+  opacity: 0.4;
+  display: inline-block;
+`
 const TableCell = ({ dataRow, dataKey, image, icon, linkTo, host, width }) => {
   let cellItem = null
   if (icon) {
       // If the column includes an icon in the cell for anndata experiment
       cellItem = dataRow[`experimentAccession`].startsWith(ANNDATA_EXPERIMENT) ?
           <div className={'icon'}>
-              <img {...icon[ANNDATA_EXPERIMENT]}/>
-              {dataRow[dataKey]}
-          </div> :
+            {dataRow[dataKey]}
+            <ExperimentIconDiv background={`red`} color={`white`} data-toggle={`tooltip`} data-placement={`bottom`}
```

Comment:
I used `indianred` in my PR.
Would you be happy to use that colour, instead of the plain `red`?
Thanks